weblate/static/multi-select.css

```diff
@@ -1,6 +1,6 @@
 .ms-container{
   background: transparent url('../img/switch.png') no-repeat 50% 50%;
-  width: 370px;
+  width: 100%;

```

Comment:
Please make the change rather in style-bootstrap.css. This file is taken verbatim from multi-select upstream and will be replaced on possible upgrades.